ptero_shell_command/implementation/celery_tasks/shell_command.py

```diff
@@ -51,16 +57,19 @@ def run(self, command_line, umask, user, working_directory,
             return exit_code == 0
 
         except PreExecFailed as e:
+            LOG.warning('pre-exec failed: %s' % e.message)
```

Comment:
I think it might be more appropriate to make this an `INFO` level message.  It's not a problem from the point of view of the service, since it probably indicates invalid client data.